src/gc/gcpriv.h

```diff
@@ -3604,6 +3604,10 @@ class gc_heap
     PER_HEAP
     size_t total_ephemeral_size;
 
+    // True if we are using allocation contexts, false if we
```

Comment:
It's true that this field is always null with Server GC but I was hoping to keep `MULTIPLE_HEAPS` defines to a minimum in the hopes of defragmenting `gc.cpp` a little bit.